WordPress/Sniffs/Theme/PluginTerritoryFunctionsSniff.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+/**
+ * Restricts the use of various functions that are plugin territory.
+ *
+ * @link    https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#regular-expressions
+ *
+ * @package WPCS\WordPressCodingStandards
+ *
+ * @since   0.xx.0
+ */
+class WordPress_Sniffs_Theme_PluginTerritoryFunctionsSniff extends WordPress_AbstractFunctionRestrictionsSniff {
+
+	/**
+	 * Groups of functions to restrict.
+	 *
+	 * Example: groups => array(
+	 * 	'lambda' => array(
+	 * 		'type'      => 'error' | 'warning',
+	 * 		'message'   => 'Use anonymous functions instead please!',
+	 * 		'functions' => array( 'eval', 'create_function' ),
+	 * 	)
+	 * )
+	 *
+	 * @return array
+	 */
+	public function getGroups() {
+		return array(
+			'plugin-territory' => array(
+				'type'      => 'error',
+				'message'   => '%s() is not allowed because it is a plugin territory function.',
```

Comment:
Just wondering if this would read better - feel free to disregard if you don't agree:
`Function %s() is not allowed because it is plugin territory`